### PR TITLE
ADD mongoRegistrationDelete info to devel manual

### PR DESCRIPTION
**Pull request by fgalan**
Doc elements of https://github.com/telefonicaid/fiware-orion/issues/3006

CC: @McMutton

### CODE REVIEW

File: doc/manuals/user/v1_v2_coexistence.md [hunk 1]
```diff
@@ -148,3 +149,9 @@ practical advantage of having more than one, from a functional point of view, e.
 [pending on a more definitive solution](https://github.com/telefonicaid/fiware-orion/issues/3044).
 
 [Top](#top)
+
+## Context availability subscriptions
+
+Note that context availability subscriptions and notifications are not included in NGSIv2. They have been intentionally left out due to it is a feature in NGSIv1 that is rarely used and doesn't worth the effort of being included in NGSIv2.
```
Comment (kzangeli) [inline doc/manuals/user/v1_v2_coexistence.md]: ... and it isn't worth the effort to include it in NGSIv2

File: doc/manuals/devel/mongoBackend.md [hunk 2]
```diff
@@ -641,6 +641,27 @@ _MB-25: mongoRegistrationCreate_
 
 [Top](#top)
 
+#### `mongoRegistrationDelete` (SR2)
+
+`mongoRegistrationDelete` encapsulates the logic for removing registrations.
+
+The header file contains only a function named `mongoRegistrationDelete()` which uses an registration ID (`regId`) as parameter.
```
Comment (kzangeli) [inline doc/manuals/devel/mongoBackend.md]: an -> a (registration ID)

File: doc/manuals/user/v1_v2_coexistence.md (hunk 1 above)
Comment (fgalan) [reply doc/manuals/user/v1_v2_coexistence.md]: Fixed in bb00131

File: doc/manuals/devel/mongoBackend.md (hunk 2 above)
Comment (fgalan) [reply doc/manuals/devel/mongoBackend.md]: Fixed in bb00131

**Pull request merged**